src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -1,41 +1,82 @@
-		- yoo
-		
-	Passive mixed content ON
-		- app, pay, passport FROM static.hdslb.com
-		- bangumi            FROM i[0-2].hdslb.com
-	Display mixed content ON
-		- app           FROM activity.hdslb.com
-		- bangumi       FROM static.hdslb.com, s[1-3].hdslb.com
-		- pay, passport FROM api*
-	* Secured by us
+		- ^bilibili.com
+		- game.bilibili.com
+		- news.bilibili.com
+		- yoo.bilibili.com
+
+	Redirect to http:
+		- live.bilibili.com
 -->
+
 <ruleset name="Bilibili.com (partial)">
+
+	<!--	Directly:	-->
 	<target host="account.bilibili.com" />
 		<test url="http://account.bilibili.com/site/nameplate.html" />
+	<target host="bml.bilibili.com" />
+		<test url="http://bml.bilibili.com/bml_intro?bid=13" />
+	<target host="bmall.bilibili.com" />
+	<target host="comment.bilibili.com" />
+		<test url="http://comment.bilibili.com/11998315.xml" />
+	<target host="interface.bilibili.com" />
+		<test url="http://interface.bilibili.com/msg.xml" />
+	<target host="pay.bilibili.com" />
+	<target host="passport.bilibili.com" />
+	<target host="static-s.bilibili.com" />
+
+	<!--	Complication:	-->
 	<target host="api.bilibili.com" />
 		<exclusion pattern="^http://api\.bilibili\.com/$" />
 		<test url="http://api.bilibili.com/x/web-show/res/loc?pf=0&#x26;id=142" />
-	<target host="bml.bilibili.com" />
-		<test url="http://bml.bilibili.com/bml_intro?bid=13" />
+
+	<target host="bangumi.bilibili.com" />
+		<exclusion pattern="^http://bangumi\.bilibili\.com/(?!sponsor/|web_api/)" />
+		<test url="http://bangumi.bilibili.com/sponsor/rankweb/get_sponsor_week_list" />
+		<test url="http://bangumi.bilibili.com/web_api/user_area" />
+
+		<test url="http://bangumi.bilibili.com/22/" />
+		<test url="http://bangumi.bilibili.com/anime/1111" />
+		<test url="http://bangumi.bilibili.com/moe/2016/jp/index" />
+
 	<target host="data.bilibili.com" />
 		<exclusion pattern="^http://data\.bilibili\.com/$" />
 		<test url="http://data.bilibili.com/a/access.js" />
-	<target host="passport.bilibili.com" />
-	<target host="pay.bilibili.com" />
-	<target host="static-s.bilibili.com" />
+
+	<target host="h.bilibili.com" />
+		<exclusion pattern="^http://h\.bilibili\.com/(?!css/|js/|images/|img/|nav\.js)" />
+		<test url="http://h.bilibili.com/css/draw_yoo.css" />
+		<test url="http://h.bilibili.com/js/picwall.js" />
+		<test url="http://h.bilibili.com/images/ra.js" />
+		<test url="http://h.bilibili.com/img/Icons2.png" />
+		<test url="http://h.bilibili.com/nav.js" />
+
+		<test url="http://h.bilibili.com/dy190003" />
+		<test url="http://h.bilibili.com/dy191437" />
+		<test url="http://h.bilibili.com/index/pic_banner.json" />
+		<test url="http://h.bilibili.com/login" />
+		<test url="http://h.bilibili.com/member?mod=addbanner&#x26;id=41" />
+
+	<target host="www.bilibili.com" />
+		<exclusion pattern="^http://www\.bilibili\.com/(?!online\.js|widget/)" />
+		<test url="http://www.bilibili.com/online.js" />
+		<test url="http://www.bilibili.com/widget/getSearchDefaultWords" />
```

Comment:
XMLHttpRequest cannot load https://www.bilibili.com/widget/getSearchDefaultWords. The 'Access-Control-Allow-Origin' header has a value 'http://bangumi.bilibili.com' that is not equal to the supplied origin. Origin 'http://www.bilibili.com' is therefore not allowed access.